lib/commands/set-default-argv.js

```diff
@@ -144,8 +146,13 @@ module.exports = function (internals, isModule) {
   internals.notransactions = internals.argv['non-transactional'];
   internals.dryRun = internals.argv['dry-run'];
   global.dryRun = internals.dryRun;
+  internals.check = internals.argv['check'];
+  global.check = internals.check;
```

Comment:
actually maybe not necessary 🤔 

why is it done for dryRun?

Reply:
It is from ancient times ;) Don't add any new global flags though, the old ones will disappear and are only still there until they get deprecated. and removed